core/src/main/java/org/geysermc/geyser/translator/protocol/java/JavaCustomPayloadTranslator.java

```diff
@@ -138,6 +139,9 @@ public void translate(GeyserSession session, ClientboundCustomPayloadPacket pack
 
                 session.sendUpstreamPacket(toSend);
             });
+        } else {
+            var eventBus = session.getGeyser().eventBus();
```

Comment:
please don't use var here

Reply:
Hey but you used var for the declare commands event 😅